crates/bitwarden-core/src/client/client.rs

```diff
@@ -66,6 +91,19 @@ impl Client {
             api_key: None,
         };
 
+        #[cfg(feature = "state")]
+        let db = {
+            let db = Database::default()
```

Comment:
Should we pass some path to the database here? Otherwise two simultaneous SDK clients will overwrite each other.